### DISCUSSION TITLE
Developer Community Call (ACP-103) 06-20-24

### DISCUSSION THREAD

**Discussion by meaghanfitzgerald** [avalanche-foundation/ACPs#111] [Announcements]
# ACP-103: Add Dynamic Fees to the X-Chain and P-Chain 06-20-2024 Meeting

Meeting Date/Time: 06-20-2024, 14:30 UTC
Meeting Duration: 1 hour
Moderator: Meaghan FitzGerald

## Purpose of the Meeting

This is a follow-up to the [ACP-83 developer call](https://github.com/avalanche-foundation/ACPs/discussions/96) held 06-05-24. [ACP-103](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/103-dynamic-fees/README.md) will introduce a dynamic fee mechanism to the X- and P-Chains.

## Who is invited?

Any protocol developer, ACP author, researcher, or Avalanche community member is invited to attend this meeting.

## Links

Sign Up Form: [Google Form](https://forms.gle/5FzarVp7QHU5KUJ16)
Youtube Link: [video]() (to be added after the meeting)

## Preparation Material

To ensure a high-quality discussion, please review the following materials prior to the meeting:

### Mandatory

ACP-103 [ReadMe](https://github.com/avalanche-foundation/ACPs/blob/main/ACPs/103-dynamic-fees/README.md)
ACP-103 [Discussion](https://github.com/avalanche-foundation/ACPs/discussions/104)

### Highly Encouraged 

[Dynamic Pricing for Non-fungible Resources](https://arxiv.org/pdf/2208.07919)
[Approaches to Charging and Rate Control](https://www.statslab.cam.ac.uk/~frank/elastic.pdf)
[Vitalik blog post on MDF](https://vitalik.eth.limo/general/2024/05/09/multidim.html)

## Agenda

Platform engineers will propose the finalized spec for adding dynamic fees to the X- and P-Chains, explaining the justification for the mechanism selected.

### Meeting Notes Summary

- To be added

**Comment by bearavax** [comment]
dynamic fees 💯
